### ISSUE TITLE
Invalid character in NUnit/JUnit summary, Azure Pipelines can't process results file

### ISSUE THREAD

**Issue by cmeeren** [haf/expecto#410]
Repro solution in case you need it: [ExpectoTest.zip](https://github.com/haf/expecto/files/6128331/ExpectoTest.zip)

When running with `--nunit-summary`, the output contains an ESC character. This causes the Azure Pipelines "Publish Test Results" task to fail:

```
##[warning]Failed to read D:\a\1\a\TestResults.xml. Error : '', hexadecimal value 0x1B, is an invalid character. Line 7, position 1.. 
```

Please see the attached test results file.

[TestResults.zip](https://github.com/haf/expecto/files/6128227/TestResults.zip)

Here is how the file looks in Notepad++. You can see the invalid character on lines 7 and 8.

![image](https://user-images.githubusercontent.com/7766733/110906333-17f96a80-830c-11eb-9bc2-4c2c587f7f00.png)

---

The same thing happens with `--junit-summary`:

```
##[warning]Failed to read D:\a\1\s\TestResults.xml. Error : '', hexadecimal value 0x1B, is an invalid character. Line 6, position 293..
```

[TestResults.zip](https://github.com/haf/expecto/files/6128239/TestResults.zip)

![image](https://user-images.githubusercontent.com/7766733/110906689-9950fd00-830c-11eb-859b-7126572c7f0b.png)

**Comment by haf** [comment]
Could you try `--colours 0`?

**Comment by cmeeren** [comment]
Yep, that solves it. I guess Expecto should always exclude colors from the test results file?

**Comment by haf** [comment]
Yes, exactly. PR? :)

**Comment by cmeeren** [comment]
Terribly sorry, I'm completely swamped and have no capacity to ramp up on a new codebase. (I have even just had to withdraw from some OSS projects I maintain.)

**Comment by cmeeren** [comment]
By the way, colors in the string is also an issue in the VS debugger (when debugging tests with F5):

![image](https://user-images.githubusercontent.com/7766733/116781602-323a0400-aa84-11eb-9e97-7a3ca3fb168b.png)
